radixdlt/src/test/java/com/radixdlt/consensus/liveness/DictatorshipTest.java

```diff
@@ -3,16 +3,15 @@
 import static org.assertj.core.api.AssertionsForClassTypes.assertThat;
 import static org.mockito.Mockito.mock;
 
-import com.radixdlt.common.EUID;
 import com.radixdlt.consensus.View;
+import com.radixdlt.crypto.ECPublicKey;
 import org.junit.Test;
 
 public class DictatorshipTest {
 	@Test
 	public void when_retrieve_dictatorship_proposer__it_should_always_be_the_same() {
-		Dictatorship dictatorship = new Dictatorship(EUID.ONE);
-		assertThat(dictatorship.getProposer(mock(View.class))).isEqualTo(EUID.ONE);
-		assertThat(dictatorship.isValidProposer(EUID.ONE, mock(View.class))).isTrue();
-		assertThat(dictatorship.isValidProposer(mock(EUID.class), mock(View.class))).isFalse();
+		ECPublicKey ecPublicKey = mock(ECPublicKey.class);
+		Dictatorship dictatorship = new Dictatorship(ecPublicKey);
+		assertThat(dictatorship.getProposer(mock(View.class))).isEqualTo(ecPublicKey);
```

Comment:
Shouldn't we check this twice (i.e. call getProposer twice) to make sure it's *actually* the always the same?